docs/faq/precision.rst

```diff
@@ -0,0 +1,44 @@
+.. doctest-skip-all
+
+.. _precision:
+
+*********
+Precision
+*********
+
+Rise/set/meridian transit calculations in `astroplan` are designed to be fast
+while achieving a precision comparable to what can be predicted given the
+affects of the changing atmosphere. As a result, there may be some slightly
+counter-intuitive behavior in `astroplan` methods like
+`astroplan.Observer.target_rise_time`, `astroplan.Observer.target_set_time` and
+`astroplan.Observer.target_meridian_transit_time`.
```

Comment:
Suggested addition:

that can lead to small (typically <1 second) changes in the numerical values of these computed timed.